src/components/navbar/mobile/parts/navmenu/SlideMenu.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { motion } from 'framer-motion';
+import userStatus from '@store/user/user-status';
+import { useStore } from '@nanostores/react';
+import ButtonsManagerM from '../buttons/ButtonsManagerM';
+
+const SlideMenu = ({ isOpen }: { isOpen: boolean }) => {
+  const { isLogged } = useStore(userStatus);
+
+  return (
+    <motion.div
+      className='fixed top-0 right-0 h-full w-48 bg-[#182852] z-[200]'
```

Comment:
Height should be relative to device size maybe w-5/12 the hit target is too small on some phones

Reply:
Also color should be put in Tailwind.css and refered to by name